cloudmailer/cloudmailer.py

```diff
@@ -241,19 +262,46 @@ def readConfiguration():
                 MAX_UPGRADE_AT_ONCE = config.get('DEFAULT', 'MAX_UPGRADE_AT_ONCE')
             break
 
-    if (not TEMPDIR or not HOST_SCHEDULE or not AFFECTED_VMS or not MAIL_SERVER or not MAIL_FROM or not MAIL_BCC):
-        print( 'You need to set all variables, TEMPDIR=%s, HOST_GROUP_DEBUG=%s,'
-               ' AFFECTED_VMS=%s, HOST_SCHEDULE=%s, MAIL_SERVER=%s, MAIL_FROM=%s '
-               % (str(TEMPDIR), str(HOST_GROUP_DEBUG), str(AFFECTED_VMS), str(HOST_SCHEDULE), str(MAIL_SERVER), str(MAIL_FROM) ))
-        print("Reading configuration failed")
-        exit(2)
+    all_present = True
+    if not TEMPDIR:
```

Comment:
TEMPDIR, HOST_SCHEDULE, AFFECTED_VMS do not necessarily need to be defined in the config file. They do have already some default values different from "False" or "None", so they will not be caught by these if statements. Also HOST_GROUP_DEBUG is printed out , but it has a default non-null value.

Reply:
This ifs cover the case of a cloudmailer.cfg file with the values empty. But this wasn't really in the scope, the change it's to show more specific information in case of error, the functionality should be intact. I'll rather move it all to click instead of working much on this anyhow.